controlplane/pkg/apis/connectioncontext/connectioncontext.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 Red Hat, Inc.
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This contains the core Network Service Mesh API definitions for external
+// consumption via gRPC protobufs.
+
+syntax = "proto3";
+
+package connectioncontext;
+option go_package = "connectioncontext";
+
+message Route {
+    string dst_ip_addr = 1;             /* destination ip address + prefix in format <address>/<prefix> */
```

Comment:
Lets call this "prefix" not "dst_ip_addr".

For the moment lets start simply with the single field "prefix".  As a general rule, all routes from an NSE  should point to the connection to the NSE.

Reply:
I supposed dst_ip_addr is a target so "prefix" is confusing for me. Maybe better it to have named "target_address"?